### PR TITLE
[xdoctest] reformat example code with google style in No. 261-263

### PR DESCRIPTION
**Pull request by kkk459**
### PR types
others

### PR changes
others

### Description
#55629

### CODE REVIEW

**Comment by megemini** [comment]
@SigureMo 对于 `use plain sample code style` 的日志，需要把 info 改为 warning ～

日志分离之后，这部分的日志没改，我先改一下 ～

----

PR：https://github.com/PaddlePaddle/Paddle/pull/57775 😅

File: python/paddle/distributed/fleet/fleet.py [hunk 1]
```diff
@@ -128,31 +128,31 @@ class Fleet:
             :name: code-example2
 
             # Example2: for parameter server training
-            import paddle
-            paddle.enable_static()
-            import paddle.distributed.fleet as fleet
-            strategy = fleet.DistributedStrategy()
-            fleet.init(strategy=strategy)
+            >>> import paddle
```
Comment (megemini) [inline python/paddle/distributed/fleet/fleet.py:131]: 示例部分要写在 `Examples:` 下面。另外，上面的 `code-example1` 也要改～

**Review by megemini — COMMENTED** [review]
先把 fleet 里面没修改的示例改一下，然后重跑 CI 看看结果吧 ～

File: python/paddle/distributed/fleet/fleet.py (hunk 1 above)
Comment (kkk459) [reply python/paddle/distributed/fleet/fleet.py:131]: done

Comment (megemini) [reply python/paddle/distributed/fleet/fleet.py:131]: `.. code-block:: python` 之前写个 `Examples:` 吧 ～ 可以参考后面的代码 ～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 2]
```diff
@@ -58,18 +58,18 @@ def fused_feedforward(
...
+        >>> residual = x
+        >>> if pre_layer_norm:
+        ...     out = layer_norm1(x)
+        >>> else:
+        ...     out = x
+        >>> out = linear2(dropout1(activation(linear1(src))))
+        >>> if add_residual:
+        ...     out = residual + dropout2(out)
+        >>> else:
+        ...     out = dropout2(out)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:70]: 这里把 `.. code-block:: python` 改为 `.. code-block:: text` 吧～

另外，`else` 属于复合语句，用 `...` 代替 `>>>`

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 3]
```diff
@@ -323,21 +324,22 @@ def fused_bias_dropout_residual_layer_norm(
...
+            >>> x = paddle.rand(shape=(2, 4, 128), dtype="float32")
+            >>> # residual: [batch_size, seq_len, embed_dim]
+            >>> residual = paddle.rand(shape=(2, 4, 128), dtype="float32")
+            >>> # linear bias: [embed_dim]
+            >>> bias = paddle.rand(shape=[128], dtype="float32")
+            >>> # output: [batch_size, seq_len, embed_dim]
+            >>> output = F.fused_bias_dropout_residual_layer_norm(
+            ...     x, residual, bias)
+            >>> # [2, 4, 128]
+            >>> print(output.shape)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:342]:
``` python

>>> print(output.shape)
(2, 4, 128)

```

需要把原有的注释方式改为输出～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 4]
```diff
@@ -495,33 +497,33 @@ def fused_multi_head_attention(
...
+        >>> # combine heads
+        >>> out = transpose(out, perm=[0, 2, 1, 3])
+        >>> # project to output
+        >>> out = linear(out)
+        >>> if add_residual:
+        ...     out = residual + dropout(out)
+        >>> else:
+        ...     out = dropout(out)
+        >>> if not pre_layer_norm:
+        ...     out = layer_norm(out)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:526]: 同样，改一下 `.. code-block:: python`

else 的提示符 ～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 5]
```diff
@@ -581,30 +583,31 @@ def fused_multi_head_attention(
...
+            >>> # self attention mask: [batch_size, num_heads, seq_len, seq_len]
+            >>> attn_mask = paddle.rand(shape=(2, 4, 4, 4), dtype="float32")
+
+            >>> # output: [batch_size, seq_len, embed_dim]
+            >>> output = F.fused_multi_head_attention(
+            ...     x, qkv_weight, linear_weight, False,
+            ...     None, None, None, None, 1e-5, qkv_bias,
+            ...     linear_bias, None, attn_mask)
+            >>> # [2, 4, 128]
+            >>> print(output.shape)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:610]: 输出改一下 ～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 6]
```diff
@@ -908,37 +911,37 @@ def fused_multi_transformer(
...
+
+        >>> residual = out;
+        >>> if pre_layer_norm:
+        ...     out = ffn_layer_norm(out)
+        >>> out = ffn1_linear(out)
+        >>> out = dropout(activation(out + ffn1_bias))
+        >>> out = ffn2_linear(out)
+        >>> out = residual + dropout(out + ffn2_bias)
+        >>> if not pre_layer_norm:
+        ...     out = ffn_layer_norm(out)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:944]: 同样，改一下 `.. code-block:: python`

else 提示符 ～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 7]
```diff
@@ -996,48 +999,49 @@ def fused_multi_transformer(
...
+            >>> attn_mask = paddle.rand(shape=(2, 1, 4, 4), dtype="float32")
+
+            >>> # output: [batch_size, seq_len, embed_dim]
+            >>> output = F.fused_multi_transformer(
+            ...     x, [ln_scale], [ln_bias], [qkv_weight], [qkv_bias],
+            ...     [linear_weight], [linear_bias], [ffn_ln_scale], [ffn_ln_bias],
+            ...     [ffn1_weight], [ffn1_bias], [ffn2_weight], [ffn2_bias],
+            ...     attn_mask=attn_mask)
+            >>> # [2, 4, 128]
+            >>> print(output.shape)
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:1044]: 输出 ～

File: python/paddle/incubate/nn/functional/fused_transformer.py [hunk 8]
```diff
@@ -290,7 +291,7 @@ def fused_bias_dropout_residual_layer_norm(
 
     .. code-block:: python
 
-        y = layer_norm(residual + dropout(bias + x))
+        >>> y = layer_norm(residual + dropout(bias + x))
```
Comment (megemini) [inline python/paddle/incubate/nn/functional/fused_transformer.py:294]: 把上面的 `.. code-block:: python` 改为 `.. code-block:: text` ～

这里应该是算法说明部分，可以不当作示例使用 ～

File: python/paddle/distributed/fleet/fleet.py [hunk 9]
```diff
@@ -1072,12 +1072,12 @@ def distributed_optimizer(self, optimizer, strategy=None):
...
-                fleet.init(is_collective=True)
-                strategy = fleet.DistributedStrategy()
-                optimizer = paddle.optimizer.SGD(learning_rate=0.001)
-                optimizer = fleet.distributed_optimizer(optimizer, strategy=strategy)
+                >>> import paddle
+                >>> import paddle.distributed.fleet as fleet
+                >>> fleet.init(is_collective=True)
+                >>> strategy = fleet.DistributedStrategy()
+                >>> optimizer = paddle.optimizer.SGD(learning_rate=0.001)
+                >>> optimizer = fleet.distributed_optimizer(optimizer, strategy=strategy)
```
Comment (megemini) [inline python/paddle/distributed/fleet/fleet.py:1080]:
``` python

>>> import paddle
>>> import paddle.distributed.fleet as fleet
>>> fleet.init(is_collective=True)
>>> strategy = fleet.DistributedStrategy()
>>> linear = paddle.nn.Linear(10, 10)
>>> optimizer = paddle.optimizer.SGD(learning_rate=0.001, parameters=linear.parameters())
>>> optimizer = fleet.distributed_optimizer(optimizer, strategy=strategy)

```

根据错误提示，应该给 optimizer 加上参数 ～ 或者用 static ～

File: python/paddle/incubate/nn/layer/fused_dropout_nd.py [hunk 10]
```diff
@@ -53,23 +53,23 @@ class FusedDropout(paddle.nn.Layer):
...
+            Tensor(shape=[2, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+                   [[2., 0., 6.],
+                    [0., 0., 0.]])
+
+            >>> m.eval()  # switch the model to test phase
+            >>> y_test = m(x)
+            >>> print(y_test)
+            Tensor(shape=[2, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+                   [[1., 2., 3.],
+                    [4., 5., 6.]])
```
Comment (megemini) [inline python/paddle/incubate/nn/layer/fused_dropout_nd.py:72]: dropout 可以试一下加 seed ～ 如果 seed 也不能固定输出的话，可以把这部分的输出 skip 包裹一下 ～

**Review by SigureMo — COMMENTED** [review]
需要根据 review 意见进行修改，并 merge 一下最新的 develop 分支

**Comment by luotao1** [comment]
@KongAKun 记得更新PR哈

**Comment by luotao1** [comment]
close due to the following PR is merged:
- https://github.com/PaddlePaddle/Paddle/pull/58168
